sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.52.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added an option to configure the minimum retry duration for 404/1002 session not available. - See [PR 37143](https://github.com/Azure/azure-sdk-for-java/pull/37143) and [PR 37240](https://github.com/Azure/azure-sdk-for-java/pull/37240)
```

Comment:
since we have changed the API name, maybe here also match the new name - `minTimeoutPerRegion`

Reply:
Will do in next PR